grails-app/domain/imod/AssessmentTechnique.groovy

```diff
@@ -21,7 +25,7 @@ class AssessmentTechnique {
 	static belongsTo = [
 		ImodUser,
 		LearningObjective,
-		//PedagogyMode
+		AssessmentFeedback

```

Comment:
`AssessmentTechnique` belongs to `AssessmentFeedback`?
So each technique has one feedback, but one feedback can be for many techniques?
I thought this would be the other way around